spotify_project/spotify/base.py

```diff
@@ -3,12 +3,17 @@
 import requests
 from albums.models import Album
 from artists.models import Artist
+from users.models import CustomUser
```

Comment:
If you are using a custom user model, it's poor form to import it directly. You should be use `get_user_model()` instead. For this app, it likely doesn't matter. But if other folks were using this code, and they had a custom user model, this would break. Whereas if you used `get_uesr_model()` it work. With magic and stuff.